README.md

```diff
@@ -4,9 +4,18 @@ Team Wander’s aim is to create a cognitive testing game which will use existin
 Team Wander aims to create an app designed for scientific research on cognitive activity. To do this, the app will contain game(s) that collect data on the user’s tendency to mind-wander. The primary game intended to be used in the app is a game that displays a stream of digits to the user. When a user sees a digit they are supposed to tap unless it is a certain predefined digit, in which case they have to ignore it. This data can hopefully be used to give insight to when people are more active and alert.
 
 ## App
-The Android application files are located in /Wander. The application was developed and can be built using Android Studio.
+The Android application files are located in 2017-Cognitive-Sensors/Wander. The application was developed and can be built using Android Studio.  In order to do this one simply clones the repository to their computer.  Then, checkout the branch that work will be done in.  Open Android Studio (make sure its fully updated or there might be build errors) and navigate to the File - > Open... -> and navigate to the Wander folder (which should have an Android Studio icon indicating that it is openable as a project) and open it.
+
+If you've never used Android Studio before there are a few things to note.  Access files from the project on the far left set of tabs under "Project" and then all the code is available, and the Java code specifically under the package that is NOT labelled (androidTest) or (test).  In order to test code you should configure a virtual phone and run it on there.  You can also put your personal android phone in developer mode and test code on there which is a lot quicker to setup.
```

Comment:
Add links here to websites that explain how to 'configure a virtual phone' and how to 'put your personal android phone in developer mode'